hibernate-core/src/main/java/org/hibernate/query/internal/QueryParameterBindingsImpl.java

```diff
@@ -317,6 +317,14 @@ public void verifyParametersBound(boolean reserveFirstParameter) {
 //		return values.toArray( new Object[values.size()] );
 	}
 
+	@Override
+	public boolean isAParameterList(QueryParameter parameter) {
+		if ( parameterListBindingMap == null ) {
+			return false;
+		}
+		return parameterListBindingMap.containsKey( parameter );
```

Comment:
Again, personally I think the following is more readable:
```
return parameterListBindingMap != null && parameterListBindingMap.containsKey( parameter );
```

Reply:
sorry but I find easier to read/understand the if statement

Reply:
This is always going to be opinion based and to a large degree location based.  For example, I personally almost always find these kind of "waterfall" boolean checks to be more readable.